src/features/fwiCalculator/FWICalculatorPage.tsx

```diff
@@ -40,16 +44,25 @@ export const FWICalculatorPage = () => {
     setStations(s)
   }
 
+  const onYearRangeChange = (timeRange: number) => {
+    setTimeRange(timeRange)
+  }
+
   const onCalculateClick = () => {
     const stationCodes = stations.map(s => s.code)
     const percentile = 90
-    const yearRange = { start: 2010, end: 2019 }
+    const yearRange = {
```

Comment:
Can you create a variable with the current year? and use it for the yearRange?
`const currYear = new Date().getFullYear()`
start: currYear - timeRange
end: currYear - 1